internal/reminder/reminder.go

```diff
@@ -143,6 +178,10 @@ func (r *reminder) sendReminders(ctx context.Context) error {
 		return fmt.Errorf("error creating reminder messages: %w", err)
 	}
 
+	if r.metrics != nil {
+		r.metrics.RecordBatch(ctx, int64(len(repos)))
+	}
```

Comment:
I could change this to a direct call, but have kept it as it is for now.

Reply:
I think the only advantage of using an indirect call is that you could hide the `if r.metrics != nil` inside the `RecordBatch` call if you use a pointer receiver.

Another option is to decide that metrics will simply always be on in Reminder -- I don't know if that will simplify the code, but I've always found the "maybe you don't want metrics" to be a strange notion in Minder that was there before I got here.

Reply:
I removed this function with a direct call. `if r.metrics != nil` is down in the code as well, so replacing it at one place breaks the consistency.